spec/features/saml/vtr_spec.rb

```diff
@@ -149,6 +153,8 @@
     expect_successful_saml_redirect
   end
 
+  scenario 'sign in with VTR request for idv includes proofed attributes'
```

Comment:
Is this missing an implementation?

Reply:
Yes it is. Willfix

Reply:
I just pushed an implementation for this scenario